lib/_http_common.js

```diff
@@ -60,7 +61,8 @@ function parserOnHeadersComplete(versionMajor, versionMinor, headers, method,
   parser.incoming = new IncomingMessage(parser.socket);
   parser.incoming.httpVersionMajor = versionMajor;
   parser.incoming.httpVersionMinor = versionMinor;
-  parser.incoming.httpVersion = versionMajor + '.' + versionMinor;
+  parser.incoming.httpVersion = httpVersion[versionMajor][versionMinor] ||
+    versionMajor + '.' + versionMinor;
```

Comment:
This was intentional to avoid jslint errors on 80 characters limit. I will indent this to be below where RHS start.